templates/CMakeLists.txt.template

```diff
@@ -97,6 +97,9 @@
   set(gRPC_PROTOBUF_PROVIDER "module" CACHE STRING "Provider of protobuf library")
   set_property(CACHE gRPC_PROTOBUF_PROVIDER PROPERTY STRINGS "module" "package")
 
+  set(gRPC_PROTOBUF_PACKAGE_TYPE "" CACHE STRING "Algorithm for searching protobuf package")
```

Comment:
Yes, this was my thinking too, but @podsvirov insisted on handling it manually. At the moment I have no idea when and on which platforms the CONFIG variant is useful, so I rely on @podsvirov 's insight.